src/test/kotlin/no/nav/syfo/behandler/kafka/dialogmeldingfrombehandler/KafkaDialogmeldingFromBehandlerSpek.kt

```diff
@@ -32,7 +43,19 @@ class KafkaDialogmeldingFromBehandlerSpek : Spek({
                     partition,
                 )
                 describe("Happy path") {
-                    it("should persist behandler from incoming sykmelding") {
+                    it("persist behandler from incoming sykmelding") {
```

Comment:
Nå gjør vel Kafka-prosesseringen bare updateDialogMeldingEnabledForPartnerId, så kanskje vi bør rename `it`-en - eller er tanken å teste mer persistering her senere?